source/_integrations/tplink.markdown

```diff
@@ -2,6 +2,7 @@
 title: TP-Link Smart Home
 description: Instructions on integrating TP-Link Smart Home Devices to Home Assistant.
 ha_category:
```

Comment:
When you update this, don't forget to update the `tplink_tapo.markdown`, too.

Reply:
Done